File-Formats/DocIO/Working-with-Footnotes-and-endnotes.md

```diff
@@ -340,6 +340,285 @@ using (WordDocument document = new WordDocument())
+   outputStream.Position = 0;
+   //Download Word document in the browser.
+   return File(outputStream, "application/msword", "Sample.docx");
+}
+{% endhighlight %}
+
+{% highlight c# tabtitle="Xamarin" %}
+using (WordDocument document = new WordDocument())
+{
+	//Create a section.
+    IWSection section = document.AddSection();
+    //Add a paragraph to a section.
+    IWParagraph paragraph = section.AddParagraph();
+    //Append the text to the paragraph.
+    paragraph.AppendText("First paragraph in First section");
+    //Append the footnote.
+    WFootnote footnote = paragraph.AppendFootnote(FootnoteType.Footnote) as WFootnote;
+    //Set the footnote character format.
+    footnote.MarkerCharacterFormat.SubSuperScript = SubSuperScript.SuperScript;
+    //Set the numbering format for the footnote.
+    document.FootnoteNumberFormat = FootEndNoteNumberFormat.Arabic;
+    //Add footnote text.
+    paragraph = footnote.TextBody.AddParagraph();
+    paragraph.AppendText("Footnote content");
+    //Set the footnote position.
+    document.FootnotePosition = FootnotePosition.PrintImmediatelyBeneathText;
+    //Add the new section to the document.
+    section = document.AddSection();
+    //Add a paragraph to a section.
+    paragraph = section.AddParagraph();
+    //Append text into the paragraph.
+    paragraph.AppendText("Paragraph in Second section.");
+    //Append the endnote.
+    WFootnote endnote = paragraph.AppendFootnote(FootnoteType.Endnote) as WFootnote;
+    //Set the endnote character format.
+    endnote.MarkerCharacterFormat.SubSuperScript = SubSuperScript.SuperScript;
+    //Set the numbering format for the endnote.
+    document.EndnoteNumberFormat = FootEndNoteNumberFormat.LowerCaseRoman;
+    //Add endnote text.
+    paragraph = endnote.TextBody.AddParagraph();
+    paragraph.AppendText("Endnote of second section");
+    //Set the endnote position
+    document.EndnotePosition = EndnotePosition.DisplayEndOfSection;
+    //Add the new section to the document.
+    section = document.AddSection();
+    //Set a section break.
+    section.BreakCode = SectionBreakCode.NoBreak;
+    //Add a new paragraph to a section.
+    paragraph = section.AddParagraph();
+    //Append text into the paragraph.
+    paragraph.AppendText("Paragraph in third Section.");
+	//Save the Word document to MemoryStream.
+	MemoryStream stream = new MemoryStream();
+	document.Save(stream, FormatType.Docx);    
+	//Save the stream as a file in the device and invoke it for viewing.
+	Xamarin.Forms.DependencyService.Get<ISave>().SaveAndView("WorkingWorddoc.docx", "application/msword", stream);
+	//Please download the helper files from the below link to save the stream as file and open the file for viewing in Xamarin platform
+    //https://help.syncfusion.com/file-formats/docio/create-word-document-in-xamarin#helper-files-for-xamarin
+}
+{% endhighlight %}
+
+{% endtabs %}  
+
+You can download a complete working sample from [GitHub]
```

Comment:
@sharonaumao  - Please add valid GitHub URL link like below,
https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Footnotes-and-Endnotes/Position-for-footnote-and-endnote